[2022-10-08T10:49:01.813+0000] {processor.py:156} INFO - Started process (PID=8294) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:01.814+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:01.815+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:01.814+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:01.816+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:01.815+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:49:01.817+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:01.834+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:49:12.683+0000] {processor.py:156} INFO - Started process (PID=8368) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:12.684+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:12.685+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:12.685+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:12.686+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:12.686+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:49:12.687+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:12.703+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:49:14.699+0000] {processor.py:156} INFO - Started process (PID=8373) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:14.699+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:14.699+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:14.699+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:14.701+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:14.700+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:49:14.702+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:14.718+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:49:24.923+0000] {processor.py:156} INFO - Started process (PID=8397) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:24.924+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:24.924+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:24.924+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:24.926+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:24.925+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:49:24.926+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:24.942+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:49:30.528+0000] {processor.py:156} INFO - Started process (PID=8452) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:30.528+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:30.529+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:30.529+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:30.531+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:30.530+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 6
    start_date=(timezone.datetime 2022, 10, )
                                     ^
SyntaxError: invalid syntax
[2022-10-08T10:49:30.531+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:30.548+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:49:31.665+0000] {processor.py:156} INFO - Started process (PID=8517) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:31.666+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:31.666+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:31.666+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:31.668+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:31.667+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 6
    start_date=(timezone.datetime 2022, 10, 8)
                                     ^
SyntaxError: invalid syntax
[2022-10-08T10:49:31.668+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:31.686+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:49:43.571+0000] {processor.py:156} INFO - Started process (PID=8591) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:43.571+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:43.572+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:43.572+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:43.574+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:43.573+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 6
    start_date=(timezone.datetime 2022, 10, 8),
                                     ^
SyntaxError: invalid syntax
[2022-10-08T10:49:43.574+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:43.589+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.020 seconds
[2022-10-08T10:49:47.600+0000] {processor.py:156} INFO - Started process (PID=8605) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:47.600+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:47.601+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:47.601+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:47.602+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:47.602+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7
    )
    ^
SyntaxError: unexpected EOF while parsing
[2022-10-08T10:49:47.603+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:47.619+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.021 seconds
[2022-10-08T10:49:49.634+0000] {processor.py:156} INFO - Started process (PID=8610) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:49.634+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:49.635+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:49.635+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:49.636+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:49.636+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 7
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:49:49.637+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:49.653+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:49:53.680+0000] {processor.py:156} INFO - Started process (PID=8615) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:53.681+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:49:53.681+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:53.681+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:53.683+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:49:53.682+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 8
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:49:53.683+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:49:53.699+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:50:05.061+0000] {processor.py:156} INFO - Started process (PID=8789) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:50:05.062+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:50:05.062+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:50:05.062+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:50:05.064+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:50:05.063+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 8
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:50:05.065+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:50:05.081+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:50:35.743+0000] {processor.py:156} INFO - Started process (PID=9001) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:50:35.744+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:50:35.745+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:50:35.744+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:50:35.747+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:50:35.745+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 8
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:50:35.747+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:50:35.764+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:51:01.707+0000] {processor.py:156} INFO - Started process (PID=9099) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:01.708+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:01.710+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:01.709+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:01.711+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:01.711+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 8
    )
    ^
SyntaxError: invalid syntax
[2022-10-08T10:51:01.712+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:01.730+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.025 seconds
[2022-10-08T10:51:02.761+0000] {processor.py:156} INFO - Started process (PID=9109) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:02.761+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:02.762+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:02.762+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:02.763+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:02.762+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 8
    ):
     ^
SyntaxError: unexpected EOF while parsing
[2022-10-08T10:51:02.763+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:02.779+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:51:13.751+0000] {processor.py:156} INFO - Started process (PID=9243) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:13.752+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:13.753+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:13.753+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:13.754+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:13.753+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    
    ^
SyntaxError: unexpected EOF while parsing
[2022-10-08T10:51:13.755+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:13.773+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.024 seconds
[2022-10-08T10:51:40.321+0000] {processor.py:156} INFO - Started process (PID=9455) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:40.322+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:40.322+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:40.322+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:40.325+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:40.325+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9, in <module>
    t1
NameError: name 't1' is not defined
[2022-10-08T10:51:40.326+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:40.342+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.024 seconds
[2022-10-08T10:51:41.373+0000] {processor.py:156} INFO - Started process (PID=9460) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:41.373+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:41.374+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:41.373+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:41.375+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:41.375+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 =
        ^
SyntaxError: invalid syntax
[2022-10-08T10:51:41.376+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:41.392+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:51:48.427+0000] {processor.py:156} INFO - Started process (PID=9465) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:48.428+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:48.428+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:48.428+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:48.432+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:48.431+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9, in <module>
    t1 = Emptyoperator
NameError: name 'Emptyoperator' is not defined
[2022-10-08T10:51:48.432+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:48.449+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.024 seconds
[2022-10-08T10:51:51.453+0000] {processor.py:156} INFO - Started process (PID=9479) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:51.453+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:51.454+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:51.454+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:51.457+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:51.456+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9, in <module>
    t1 = Emptyoperator(task_)
NameError: name 'Emptyoperator' is not defined
[2022-10-08T10:51:51.457+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:51.474+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.024 seconds
[2022-10-08T10:51:53.515+0000] {processor.py:156} INFO - Started process (PID=9484) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:53.515+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:53.516+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:53.516+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:53.519+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:53.518+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9, in <module>
    t1 = Emptyoperator(task_id)
NameError: name 'Emptyoperator' is not defined
[2022-10-08T10:51:53.519+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:53.536+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:51:57.551+0000] {processor.py:156} INFO - Started process (PID=9489) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:57.552+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:51:57.552+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:57.552+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:57.554+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:51:57.553+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 = Emptyoperator(task_id't1')
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:51:57.554+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:51:57.571+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:52:08.125+0000] {processor.py:156} INFO - Started process (PID=9663) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:08.126+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:08.126+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:08.126+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:08.128+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:08.127+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 = Emptyoperator(task_id"t1')
                                  ^
SyntaxError: EOL while scanning string literal
[2022-10-08T10:52:08.128+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:08.144+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:52:10.309+0000] {processor.py:156} INFO - Started process (PID=9688) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:10.309+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:10.310+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:10.310+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:10.312+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:10.311+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 = Emptyoperator(task_id"t1")
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:52:10.312+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:10.328+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:52:18.369+0000] {processor.py:156} INFO - Started process (PID=9702) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:18.369+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:18.370+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:18.370+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:18.371+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:18.371+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 = Emptyoperator(task_id"t1")
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:52:18.372+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:18.388+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:52:19.384+0000] {processor.py:156} INFO - Started process (PID=9707) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:19.385+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:19.385+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:19.385+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:19.387+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:19.386+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 = Emptyoperator(task_id"t1") #
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:52:19.387+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:19.405+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:52:26.447+0000] {processor.py:156} INFO - Started process (PID=9721) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:26.448+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:26.448+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:26.448+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:26.450+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:26.449+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 = Emptyoperator(task_id"t1") #ตั้งชื่อฟังก์ชันชื่อเดียวกับไฟล์
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:52:26.450+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:26.466+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.021 seconds
[2022-10-08T10:52:29.478+0000] {processor.py:156} INFO - Started process (PID=9726) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:29.479+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:29.479+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:29.479+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:29.481+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:29.480+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 = Emptyoperator(task_id"t1") #ตั้งชื่อชื่อเดียวกับไฟล์
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:52:29.481+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:29.498+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:52:35.268+0000] {processor.py:156} INFO - Started process (PID=9800) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:35.269+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:35.269+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:35.269+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:35.271+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:35.270+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    t1 = Emptyoperator(task_id"t1") #ตั้งชื่อชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:52:35.272+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:35.290+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.025 seconds
[2022-10-08T10:52:44.147+0000] {processor.py:156} INFO - Started process (PID=9925) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:44.147+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:44.148+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:44.148+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:44.149+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:44.149+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 2
    from airflow
                ^
SyntaxError: invalid syntax
[2022-10-08T10:52:44.150+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:44.165+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.021 seconds
[2022-10-08T10:52:47.171+0000] {processor.py:156} INFO - Started process (PID=9939) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:47.171+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:47.172+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:47.172+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:47.173+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:47.173+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 2
    from airflow
               ^
SyntaxError: invalid syntax
[2022-10-08T10:52:47.174+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:47.190+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:52:58.245+0000] {processor.py:156} INFO - Started process (PID=9953) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:58.246+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:52:58.247+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:58.247+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:58.248+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:52:58.248+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 2
    from airflow.operator
                        ^
SyntaxError: invalid syntax
[2022-10-08T10:52:58.249+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:52:58.264+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.021 seconds
[2022-10-08T10:53:08.600+0000] {processor.py:156} INFO - Started process (PID=10107) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:08.601+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:53:08.601+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:08.601+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:08.603+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:08.602+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 2
    from airflow.operators.emp
                             ^
SyntaxError: invalid syntax
[2022-10-08T10:53:08.603+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:08.677+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.079 seconds
[2022-10-08T10:53:10.891+0000] {processor.py:156} INFO - Started process (PID=10152) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:10.891+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:53:10.892+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:10.892+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:10.894+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:10.893+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 2
    from airflow.operators.empty
                                ^
SyntaxError: invalid syntax
[2022-10-08T10:53:10.894+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:10.911+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:53:20.939+0000] {processor.py:156} INFO - Started process (PID=10166) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:20.940+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:53:20.940+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:20.940+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:20.942+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:20.941+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 2
    from airflow.operators.empty import
                                       ^
SyntaxError: invalid syntax
[2022-10-08T10:53:20.942+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:20.958+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:53:26.992+0000] {processor.py:156} INFO - Started process (PID=10171) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:26.992+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:53:26.993+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:26.993+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:26.994+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:26.994+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = Emptyoperator(task_id"t1") #ตั้งชื่อชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:53:26.995+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:27.011+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.021 seconds
[2022-10-08T10:53:34.500+0000] {processor.py:156} INFO - Started process (PID=10235) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:34.501+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:53:34.501+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:34.501+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:34.503+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:34.502+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = Emptyoperator(task_id"t1") #ตั้งชื่อชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:53:34.503+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:34.521+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.024 seconds
[2022-10-08T10:53:37.400+0000] {processor.py:156} INFO - Started process (PID=10293) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:37.400+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:53:37.401+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:37.401+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:37.403+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:53:37.402+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:53:37.403+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:53:37.419+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:54:07.738+0000] {processor.py:156} INFO - Started process (PID=10490) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:07.739+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:54:07.739+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:07.739+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:07.741+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:07.740+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:54:07.741+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:07.762+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.027 seconds
[2022-10-08T10:54:26.095+0000] {processor.py:156} INFO - Started process (PID=10596) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:26.095+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:54:26.096+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:26.096+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:26.098+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:26.097+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:54:26.098+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:26.114+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.021 seconds
[2022-10-08T10:54:28.114+0000] {processor.py:156} INFO - Started process (PID=10601) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:28.115+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:54:28.115+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:28.115+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:28.118+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:28.116+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:54:28.118+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:28.134+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:54:37.894+0000] {processor.py:156} INFO - Started process (PID=10675) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:37.894+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:54:37.895+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:37.895+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:37.897+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:37.896+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:54:37.897+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:37.915+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.024 seconds
[2022-10-08T10:54:41.889+0000] {processor.py:156} INFO - Started process (PID=10809) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:41.890+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:54:41.890+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:41.890+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:41.892+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:41.891+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:54:41.892+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:41.968+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.082 seconds
[2022-10-08T10:54:46.008+0000] {processor.py:156} INFO - Started process (PID=10814) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:46.009+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:54:46.009+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:46.009+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:46.011+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:54:46.010+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:54:46.011+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:54:46.028+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:55:16.939+0000] {processor.py:156} INFO - Started process (PID=11026) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:55:16.940+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:55:16.941+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:55:16.940+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:55:16.942+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:55:16.941+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:55:16.943+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:55:16.959+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:55:47.674+0000] {processor.py:156} INFO - Started process (PID=11229) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:55:47.675+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:55:47.675+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:55:47.675+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:55:47.677+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:55:47.676+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:55:47.677+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:55:47.694+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:56:18.074+0000] {processor.py:156} INFO - Started process (PID=11441) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:18.074+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:56:18.075+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:18.075+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:18.077+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:18.076+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:56:18.077+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:18.093+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:56:48.858+0000] {processor.py:156} INFO - Started process (PID=11653) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:48.859+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:56:48.859+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:48.859+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:48.861+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:48.860+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 10
    t1 = EmptyOperator(task_id"t1") #ตั้งชื่อเดียวกับinstance
                                 ^
SyntaxError: invalid syntax
[2022-10-08T10:56:48.862+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:48.878+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:56:57.920+0000] {processor.py:156} INFO - Started process (PID=11667) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:57.921+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:56:57.922+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:57.922+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:57.934+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:56:58.050+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:58.050+0000] {manager.py:507} INFO - Created Permission View: can edit on DAG:my_dag
[2022-10-08T10:56:58.058+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:58.058+0000] {manager.py:507} INFO - Created Permission View: can delete on DAG:my_dag
[2022-10-08T10:56:58.064+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:58.063+0000] {manager.py:507} INFO - Created Permission View: can read on DAG:my_dag
[2022-10-08T10:56:58.064+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:58.064+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:56:58.077+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:58.077+0000] {dag.py:2594} INFO - Creating ORM DAG for my_dag
[2022-10-08T10:56:58.087+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:56:58.087+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:56:58.102+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.184 seconds
[2022-10-08T10:57:03.145+0000] {processor.py:156} INFO - Started process (PID=11672) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:57:03.145+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:57:03.146+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:57:03.146+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:57:03.157+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:57:03.180+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:57:03.180+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:57:03.204+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:57:03.204+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:57:03.220+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.078 seconds
[2022-10-08T10:57:33.431+0000] {processor.py:156} INFO - Started process (PID=11884) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:57:33.431+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:57:33.432+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:57:33.432+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:57:33.442+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:57:33.476+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:57:33.476+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:57:33.500+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:57:33.500+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:57:33.516+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.088 seconds
[2022-10-08T10:58:04.059+0000] {processor.py:156} INFO - Started process (PID=12088) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:04.060+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:58:04.061+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:04.060+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:04.071+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:04.106+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:04.106+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:58:04.132+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:04.132+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:58:04.148+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.091 seconds
[2022-10-08T10:58:34.173+0000] {processor.py:156} INFO - Started process (PID=12300) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:34.173+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:58:34.174+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:34.174+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:34.185+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:34.221+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:34.221+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:58:34.246+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:34.245+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:58:34.262+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.092 seconds
[2022-10-08T10:58:39.955+0000] {processor.py:156} INFO - Started process (PID=12353) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:39.955+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:58:39.956+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:39.956+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:39.970+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:40.007+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:40.007+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:58:40.033+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:40.032+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:58:40.050+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.098 seconds
[2022-10-08T10:58:42.085+0000] {processor.py:156} INFO - Started process (PID=12374) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:42.085+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:58:42.086+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:42.086+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:42.088+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:42.087+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    tag
    ^
SyntaxError: positional argument follows keyword argument
[2022-10-08T10:58:42.088+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:42.106+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:58:59.003+0000] {processor.py:156} INFO - Started process (PID=12513) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:59.004+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:58:59.004+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:59.004+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:59.008+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:58:59.008+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9, in <module>
    tags = workshop
NameError: name 'workshop' is not defined
[2022-10-08T10:58:59.009+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:58:59.023+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:59:01.023+0000] {processor.py:156} INFO - Started process (PID=12518) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:01.023+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:01.024+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:01.023+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:01.025+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:01.025+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    tags = workshop]
                   ^
SyntaxError: invalid syntax
[2022-10-08T10:59:01.026+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:01.042+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.022 seconds
[2022-10-08T10:59:03.087+0000] {processor.py:156} INFO - Started process (PID=12525) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:03.087+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:03.088+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:03.087+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:03.091+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:03.091+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9, in <module>
    tags = [workshop]
NameError: name 'workshop' is not defined
[2022-10-08T10:59:03.092+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:03.108+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.024 seconds
[2022-10-08T10:59:10.719+0000] {processor.py:156} INFO - Started process (PID=12572) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:10.720+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:10.721+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:10.721+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:10.723+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:10.722+0000] {dagbag.py:330} ERROR - Failed to import: /opt/airflow/dags/my_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 326, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/my_dag.py", line 9
    tags = ["workshop]
                     ^
SyntaxError: EOL while scanning string literal
[2022-10-08T10:59:10.724+0000] {processor.py:770} WARNING - No viable dags retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:10.740+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.023 seconds
[2022-10-08T10:59:13.147+0000] {processor.py:156} INFO - Started process (PID=12600) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:13.148+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:13.148+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:13.148+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:13.160+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:13.233+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:13.233+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:59:13.257+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:13.257+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:59:13.276+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.131 seconds
[2022-10-08T10:59:14.969+0000] {processor.py:156} INFO - Started process (PID=12639) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:14.970+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:14.971+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:14.970+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:14.986+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:15.081+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:15.080+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:59:15.165+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:15.165+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:59:15.184+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.218 seconds
[2022-10-08T10:59:16.296+0000] {processor.py:156} INFO - Started process (PID=12679) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:16.297+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:16.297+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:16.297+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:16.363+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:16.388+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:16.387+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:59:16.465+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:16.465+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:59:16.483+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.190 seconds
[2022-10-08T10:59:27.444+0000] {processor.py:156} INFO - Started process (PID=12755) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:27.445+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:27.445+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:27.445+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:27.457+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:27.495+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:27.495+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:59:27.521+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:27.521+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:59:27.543+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.102 seconds
[2022-10-08T10:59:28.472+0000] {processor.py:156} INFO - Started process (PID=12760) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:28.473+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:28.473+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:28.473+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:28.484+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:28.508+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:28.508+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:59:28.535+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:28.535+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:59:28.553+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.084 seconds
[2022-10-08T10:59:59.446+0000] {processor.py:156} INFO - Started process (PID=12963) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:59.447+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T10:59:59.447+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:59.447+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:59.459+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T10:59:59.492+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:59.492+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T10:59:59.517+0000] {logging_mixin.py:117} INFO - [2022-10-08T10:59:59.517+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T10:59:59.533+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.089 seconds
[2022-10-08T11:00:30.311+0000] {processor.py:156} INFO - Started process (PID=13175) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:00:30.312+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:00:30.312+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:00:30.312+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:00:30.324+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:00:30.358+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:00:30.358+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:00:30.383+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:00:30.383+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:00:30.399+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.091 seconds
[2022-10-08T11:01:00.644+0000] {processor.py:156} INFO - Started process (PID=13387) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:01:00.645+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:01:00.645+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:01:00.645+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:01:00.657+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:01:00.695+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:01:00.695+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:01:00.721+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:01:00.721+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:01:00.738+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.097 seconds
[2022-10-08T11:01:30.988+0000] {processor.py:156} INFO - Started process (PID=13590) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:01:30.989+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:01:30.989+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:01:30.989+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:01:31.004+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:01:31.043+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:01:31.042+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:01:31.067+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:01:31.067+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:01:31.084+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.098 seconds
[2022-10-08T11:02:01.591+0000] {processor.py:156} INFO - Started process (PID=13802) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:02:01.592+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:02:01.592+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:02:01.592+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:02:01.604+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:02:01.637+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:02:01.637+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:02:01.662+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:02:01.661+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:02:01.679+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.090 seconds
[2022-10-08T11:02:32.428+0000] {processor.py:156} INFO - Started process (PID=14014) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:02:32.429+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:02:32.429+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:02:32.429+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:02:32.441+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:02:32.483+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:02:32.483+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:02:32.508+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:02:32.508+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:02:32.524+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.099 seconds
[2022-10-08T11:03:03.363+0000] {processor.py:156} INFO - Started process (PID=14217) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:03:03.364+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:03:03.365+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:03:03.364+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:03:03.376+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:03:03.410+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:03:03.410+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:03:03.435+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:03:03.435+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:03:03.451+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.090 seconds
[2022-10-08T11:03:33.767+0000] {processor.py:156} INFO - Started process (PID=14430) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:03:33.768+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:03:33.769+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:03:33.768+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:03:33.779+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:03:33.815+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:03:33.814+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:03:33.839+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:03:33.839+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:03:33.857+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.092 seconds
[2022-10-08T11:04:04.629+0000] {processor.py:156} INFO - Started process (PID=14642) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:04:04.630+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:04:04.630+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:04:04.630+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:04:04.641+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:04:04.674+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:04:04.674+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:04:04.699+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:04:04.699+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:04:04.715+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.089 seconds
[2022-10-08T11:04:35.283+0000] {processor.py:156} INFO - Started process (PID=14845) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:04:35.284+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:04:35.284+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:04:35.284+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:04:35.295+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:04:35.328+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:04:35.328+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:04:35.352+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:04:35.352+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:04:35.368+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.088 seconds
[2022-10-08T11:05:05.756+0000] {processor.py:156} INFO - Started process (PID=15057) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:05.757+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:05:05.757+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:05.757+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:05.768+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:05.802+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:05.802+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:05:05.829+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:05.829+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:05:05.846+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.093 seconds
[2022-10-08T11:05:27.596+0000] {processor.py:156} INFO - Started process (PID=15260) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:27.597+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:05:27.597+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:27.597+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:27.609+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:27.674+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:27.673+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:05:27.699+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:27.699+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:05:27.717+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.124 seconds
[2022-10-08T11:05:30.640+0000] {processor.py:156} INFO - Started process (PID=15265) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:30.640+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:05:30.641+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:30.641+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:30.653+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:30.692+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:30.692+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:05:30.718+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:30.718+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:05:30.736+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.098 seconds
[2022-10-08T11:05:35.777+0000] {processor.py:156} INFO - Started process (PID=15279) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:35.777+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:05:35.778+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:35.778+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:35.788+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:35.822+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:35.822+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:05:35.845+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:35.845+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:05:35.863+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.088 seconds
[2022-10-08T11:05:41.812+0000] {processor.py:156} INFO - Started process (PID=15284) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:41.813+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:05:41.813+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:41.813+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:41.824+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:41.900+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:41.900+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:05:41.925+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:41.925+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:05:41.944+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.135 seconds
[2022-10-08T11:05:46.940+0000] {processor.py:156} INFO - Started process (PID=15298) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:46.941+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:05:46.941+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:46.941+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:46.954+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:05:46.978+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:46.978+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:05:47.004+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:05:47.004+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:05:47.024+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.086 seconds
[2022-10-08T11:06:17.948+0000] {processor.py:156} INFO - Started process (PID=15501) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:06:17.949+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:06:17.950+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:06:17.949+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:06:17.961+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:06:18.010+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:06:18.010+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:06:18.037+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:06:18.037+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:06:18.054+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.108 seconds
[2022-10-08T11:06:48.772+0000] {processor.py:156} INFO - Started process (PID=15713) to work on /opt/airflow/dags/my_dag.py
[2022-10-08T11:06:48.772+0000] {processor.py:758} INFO - Processing file /opt/airflow/dags/my_dag.py for tasks to queue
[2022-10-08T11:06:48.773+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:06:48.773+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/airflow/dags/my_dag.py
[2022-10-08T11:06:48.785+0000] {processor.py:768} INFO - DAG(s) dict_keys(['my_dag']) retrieved from /opt/airflow/dags/my_dag.py
[2022-10-08T11:06:48.835+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:06:48.835+0000] {dag.py:2573} INFO - Sync 1 DAGs
[2022-10-08T11:06:48.861+0000] {logging_mixin.py:117} INFO - [2022-10-08T11:06:48.861+0000] {dag.py:3328} INFO - Setting next_dagrun for my_dag to None, run_after=None
[2022-10-08T11:06:48.878+0000] {processor.py:178} INFO - Processing /opt/airflow/dags/my_dag.py took 0.109 seconds
